src/components/timeout-dialog/timeout-dialog.test.js

```diff
@@ -80,23 +79,23 @@ describe('/components/timeout-dialog', () => {
       currentDateTime: 1554196031049, // the time these tests were written
       // - this can change but it's best not to write randomness into tests
     };
-    mock.spyOn(Date, 'now').mockImplementation(() => testScope.currentDateTime);
-    mock.spyOn(utils, 'ajaxGet').mockImplementation(() => {
+    jest.spyOn(Date, 'now').mockImplementation(() => testScope.currentDateTime);
+    jest.spyOn(utils, 'ajaxGet').mockImplementation(() => {
     });
-    mock.spyOn(redirectHelper, 'redirectToUrl').mockImplementation(() => {
+    jest.spyOn(redirectHelper, 'redirectToUrl').mockImplementation(() => {
     });
-    mock.spyOn(dialog, 'displayDialog').mockImplementation(($elementToDisplay) => {
+    jest.spyOn(dialog, 'displayDialog').mockImplementation(($elementToDisplay) => {
       testScope.latestDialog$element = $elementToDisplay;
       testScope.latestDialogControl = {
-        closeDialog: mock.fn(),
-        setAriaLabelledBy: mock.fn(),
-        addCloseHandler: mock.fn().mockImplementation((fn) => {
+        closeDialog: jest.fn(),
+        setAriaLabelledBy: jest.fn(),
+        addCloseHandler: jest.fn().mockImplementation((fn) => {
           testScope.latestDialogCloseCallback = fn;
         }),
       };
       return testScope.latestDialogControl;
     });
-    jest.useFakeTimers();
+    jest.useFakeTimers('legacy');
```

Comment:
jest 27 changed default behaviour, this gets us the legacy behaviours our tests depend on (useFakeTimers mocks things like window.setTimeout)